internal/impl/kube.go

```diff
@@ -766,7 +785,7 @@ func newDeployment(app *protos.AppConfig, cfg *kubeConfig, depId, image string)
 }
 
 // newListener returns a new listener.
-func newListener(depId string, config *kubeConfig, name string) listener {
+func newListener(depId string, config *kubeConfig, name string) (listener, error) {
```

Comment:
I don't think this function ever returns a non-nil error. Maybe we can just return a `listener` without the `error`?

Reply:
Good catch